app/controllers/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def index
+    @users = User.order('id ASC').page(params[:page]).per(1)
```

Comment:
```suggestion
    @users = User.order(:id).page(params[:page]).per(1)
```

でいいですね。
あとper(1) は動作確認用にはいいですが、実際のアプリケーションではふつうやらないので、見直してみてほしいですー。